wandb/sync.py

```diff
@@ -117,7 +117,7 @@ def watch(self, files):
                 ), {"diff.patch": open(__stage_dir__+"diff.patch", "rb")})
             self._observer.start()
             print("Syncing %s" % self.url)
-            # Piped mode TODO: remove this gimic
+            # TODO: put the sub process run logic here?
```

Comment:
I didn't rip out the `source_proc` pipe logic, figured it might be useful for the sub process.